rust/android/Dockerfile

```diff
@@ -81,22 +83,22 @@ RUN rustup component add rustfmt
 RUN cd /opt && git clone --depth=1 https://github.com/zecwalletco/zecwallet-mobile zecwalletmobile && cd zecwalletmobile/rust/android && cargo fetch && cargo build --release && rm -rf /opt/zecwalletmobile
```

Comment:
Is this the URL we want to be cloning from?

Reply:
No...   but I intended to address that in a separate PR..